kythe/typescript/testdata/namespace.ts

```diff
@@ -17,3 +17,13 @@ namespace N {
 //- @N ref NValue
 //- @obj ref NObj
 N.obj;
+
+// Repeat same namespace to make
+// sure we don't output duplicate
+// facts.
+//- SecondNdef defines NValue
+//- SecondNdef.loc/start @^"namespace"
+namespace N {
+	export const boo = 1;
+  //- SecondNdef.loc/end @$"}"
+}
```

Comment:
What about something like:

```
namespace N.z {
}
```

Do we emit `N` again here?

Reply:
Yes. We emit `defines` edge but not `defines/binding`. Added test for N.Z.